### ISSUE TITLE
Test: a performance test for Llama models

### ISSUE THREAD

**Issue by Muxas** [nntile/nntile#127]
It shall be a separate script launched manually. Main goal is to compare forward, forward+backward, gradient accumulation (several forward+backward) with an optimizer step (AdamW and SGD) of NNTile against PyTorch in terms of tokens/sec, MFU (model flops utilisation) and HFU (hardware flops utilisation).

**Comment by Muxas** [comment]
This is partially done by #197
